src/main/resources/assets/js/app/browse/ApplicationTreeGrid.ts

```diff
@@ -15,15 +15,17 @@ import {ListApplicationsRequest} from 'lib-admin-ui/application/ListApplications
 import {Body} from 'lib-admin-ui/dom/Body';
 import {UploadItem} from 'lib-admin-ui/ui/uploader/UploadItem';
 
+declare const CONFIG;
+
 export class ApplicationTreeGrid extends TreeGrid<Application> {
 
     constructor() {
-        const builder = new TreeGridBuilder<Application>().setColumnConfig([{
+        const builder = new TreeGridBuilder<Application>().setCheckableRows(false).setColumnConfig([{
```

Comment:
With this the checkbox column will never show, even when readonly mode is off.

![image](https://user-images.githubusercontent.com/11612490/97869812-58b7f600-1d12-11eb-9a52-7c605a84d7cd.png)